service-front/module/Application/view/application/pages/cp/choose_country.twig

```diff
@@ -31,45 +31,22 @@
                             {{ form.get('country').messages | join(', ', ' and ') }}
                         </p>
                     {% endif %}
-                        <div class="govuk-form-group" id="choose-country">
-                            <label class="govuk-label" for="sort">
+                    <div class="govuk-form-group" id="choose-country">
+                        <label class="govuk-label" for="sort">
 
-                            </label>
-                            <select class="govuk-select" id="country" name="country">
-                                <option value=""></option>
-                                {% for code, name in countries_data %}
-                                    <option
+                        </label>
+                        <select class="govuk-select" id="country" name="country">
```

Comment:
This label should have content, and should be `for` an element that exists on the page ("country"?)